data/lenet_memory_solver.prototxt

```diff
@@ -1,9 +1,9 @@
 # The train/test net protocol buffer definition
 net: "lenet_memory_train_test.prototxt"
 # test_iter specifies how many forward passes the test should carry out.
-test_iter: 1
+test_iter: 2
```

Comment:
We should not use this solver file in data folder. Unit tests etc should use the solver file in test/resources folder.